packages/effect/src/PubSub.ts

```diff
@@ -46,7 +46,10 @@ export interface PubSub<in out A> extends Queue.Enqueue<A>, Pipeable {
  * @since 2.0.0
  * @category constructors
  */
-export const bounded: <A>(requestedCapacity: number) => Effect.Effect<PubSub<A>> = internal.bounded
+export const bounded: <A>(
+  requestedCapacity: number,
+  options?: { readonly replayCapacity?: number }
+) => Effect.Effect<PubSub<A>> = internal.bounded
```

Comment:
Maybe it makes sense to implement like that?
```ts
export const bounded: <A>(
  capacity: number | {capacity: number; replay?: number},
) => Effect.Effect<PubSub<A>> = internal.bounded
```

Because the usage looks non-consistent

```ts
PubSub.bounded(5, {replayCapacity: 10})
```

in comparison with

```ts
PubSub.bounded({capacity: 5, replay: 10})
```

Reply:
Yeah that might be better.